wiktionaryparser.py

```diff
@@ -251,9 +257,10 @@ def map_to_object(self, word_data):
             json_obj_list.append(data_obj.to_json())
         return json_obj_list
 
-    def fetch(self, word, language=None):
+    def fetch(self, word, language=None, oldid=None):
         language = self.language if not language else language
-        response = self.session.get(self.url.format(word))
+        url_ = self.get_url(word, oldid)
+        response = self.session.get(url_)
```

Comment:
I think passing URL params as a dictionary is a better way, as mentioned [here](http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls). So when oldId is `None` it won't be passed at all and `get_url()` wouldn't be needed

Reply:
I didn't know about it, i'll modify as you suggested.